.github/workflows/release.yaml

```diff
@@ -61,19 +61,19 @@ jobs:
       - lib-check
       - lint
       - unit-test
-      - integration-test-lxd
+      - integration-test
```

Comment:
Copy/paste?

Reply:
shameless copy paste, thanks for catching this 🤦‍♀️